packages/hardhat-ethers/test/index.ts

```diff
@@ -406,11 +406,11 @@ describe("Ethers plugin", function () {
                 },
               });
             } catch (reason: any) {
```

Comment:
why in a different place did you have to change the caught object's type to `unknown` but here you left it as `any`?

Reply:
I don't think we have a well-defined guideline for this (yet?). In general, I think we should try to use `unknown` if possible, but I guess that here I didn't want to modify the code too much because this is a critical plugin.